SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security
+
+This document specifies the CVE (Common Vulnerabilities and Exposure) process
```

Comment:
maybe just "security process"? We will issue CVEs but not sure if it's called the CVE process.